### DISCUSSION TITLE
你对第一次的活动有什么感想？

### DISCUSSION THREAD

**Discussion by suredream** [suredream/hack-your-ds-interview#13] [General]
欢迎任何的反馈，以及对以后活动的建议，也可以完成下面的 survey

1. 你对于活动的内容 [0-没有学到，1-一般，2-很好]
2. 活动的长度 [0-太短，1-太长，2-合理]
3. 话题的深度 [0-太浅，1-太深，2-合理]
4. 活动的节奏 [0-太慢，1-太快，2-合理]
5. 参与度 [0-难以参与，1-想参与但无方式，2-暂无感觉]

**Comment by JJ-Zhang-DS** [comment]
你对于活动的内容 [2-很好]
活动的长度 [合理]
活题的深度� [合理]
活动的节奏� [合理]
参与度[1-想参与但无方式]

作为一个初学者，个人感觉，是不是在最前面加一个abstract，简明扼要的总结一下：背景（任务）、方法、结果、结论（展望）。
这次会前没有review notebook，这样参与度就很难很高。如果能在很短的时间内了解一下整个project，是不是可以更快的找到切入点。
我觉得如果能看到real DS的工作或者他们对notebook的评价，应该对初学者帮助更大。
非常感谢您的组织协调，相信会吸引越来越多的人参与进来。

**Reply by suredream** [reply]
是的，第一次大家可能都会有这种感觉，但是我们的想法就是开会之前自己去 review notebook。因为时间是有限的。
但是为 notebook 生成更好的文档会有很大的帮助，这点会想办法改进。

**Comment by lihaixia513** [comment]
我覺得各方面都合理，跟上面朋友感覺一樣，如果提前能對project有個框架介紹更好。可否简单给出review notebook的建议？我觉得可能是review下整体project的思路和技术处理（code）。

**Reply by suredream** [reply]
好，我们会准备一小段介绍式文字。review book 一开始是开放的，任何针对文档和 code 的意见都可以。活动举行一段时间后，根据实际经验，我们会组织大家来编写 review 的 guidance。
